meetings/21-07-2023.md

```diff
@@ -0,0 +1,54 @@
+# DACCS executive committee meeting
+
+## Logistics
+
+Where: Online
+
+When: 21/07/2023
+
+Committee members present:
+
+Other attendees:
+
+## Agenda
+
+1. Round Table
+2. Software Architecture
+   - Minimal subset of components for a node
+     - https://github.com/DACCS-Climate/DACCS-executive-committee/issues/1
+3. Potential Risks
+4. Federation Decision
+5. Opportunities for Growth
+6. Other Topics
+   - DACCS Network name
+     - https://github.com/DACCS-Climate/DACCS-executive-committee/issues/2
+     - https://github.com/bird-house/birdhouse-deploy/issues/357
+
+## Action Items
+
+
+## Notes
+
+1. Round Table
+2. Software Architecture
+3. Potential Risks
+4. Federation Decision
+5. Opportunities for Growth
+6. Other Topics
+
+## Committee Decisions
+
+1. Description: What should the name of the DACCS network be?
+   
+   Vote count:
+      Marble:
+      Glacier (Global Analytics for Climate Information Exploration and Research):
```

Comment:
FYI Glacier is also the name of a very cheap but slow storage on Amazon: https://aws.amazon.com/s3/storage-classes/glacier/

Reply:
❗ one more reason to drop that proposal then